frontend/src/routes/_app.tsx

```diff
@@ -0,0 +1,9 @@
+import { createFileRoute, redirect } from "@tanstack/react-router";
+
+export const Route = createFileRoute("/_app")({
+  beforeLoad() {
+    if (process.env.NODE_ENV !== "local") {
```

Comment:
Better use Vite env vars maybe? Not sure what the diff is.

Reply:
Maybe in another PR? Not related to this change - we've been using `process.env.NODE_ENV` for a while

Reply:
Idk look into it at least, make sure there is no problematic difference. Doesn't hurt to double check this